model-api-gen-gradle/src/main/kotlin/org/modelix/metamodel/gradle/GenerateAntScriptForMpsMetaModelExport.kt

```diff
@@ -101,8 +98,8 @@ abstract class GenerateAntScriptForMpsMetaModelExport @Inject constructor(of: Ob
         """.trimIndent())
     }
 
-    private fun getMpsBuildPropertiesFile() = mpsHome.get().asFile.resolve("build.properties")
-    private fun getMpsLanguagesDir() = mpsHome.get().asFile.resolve("languages")
+    private fun getMpsBuildPropertiesFile() = File(mpsHome.get()).resolve("build.properties")
+    private fun getMpsLanguagesDir() = File(mpsHome.get()).resolve("languages")
```

Comment:
Why are these changes necessary?

Reply:
That was necessary, because `mpsHome` became as `Property<String>` instead of a `DirectoryProperty`,
`DirectoryProperty` can not be used with  the `InputDirectory` annotation.

Reply:
Right, overlooked that.